### ISSUE TITLE
Tensor.softmax() wasn't include in documentation

### ISSUE THREAD

**Issue by salykovaa** [pytorch/pytorch#91107]
### 📚 The doc issue

Hi pytorch developers!

I've just noticed that Tensors have softmax() method, but it wasn't include in documentation. Is there any reason why it is missing? 

E.g. ``` torch.rand(2, 2).softmax(-1) ```

cc @svekars @carljparker

**Comment by lezcano** [comment]
This is an oversight. We would accept a PR fixing this.

**Comment by ShisuiUzumaki** [comment]
> This is an oversight. We would accept a PR fixing this.

I would like to work on this @lezcano What would be the appropriate method/location for/of adding it

**Comment by albanD** [comment]
Hey!
You can follow the example of `torch.tanh` for example: https://github.com/pytorch/pytorch/blob/29b119d04d1290205f75de5be3fb0f42a69ce5b1/torch/_torch_docs.py#L11501 and  https://github.com/pytorch/pytorch/blob/29b119d04d1290205f75de5be3fb0f42a69ce5b1/torch/_tensor_docs.py#L5279

**Comment by lezcano** [comment]
then also adding the relevant entry in the `docs/` folder as well. Consider using `git grep` to find other similar functions and follow the way they do it.

**Comment by ShisuiUzumaki** [comment]
> then also adding the relevant entry in the `docs/` folder as well. Consider using `git grep` to find other similar functions and follow the way they do it.

By `docs/` folder you mean?

**Comment by lezcano** [comment]
`cd docs; git grep tanh`

**Comment by ShisuiUzumaki** [comment]
> `cd docs; git grep tanh`

Oh, ok, thanks for the advice! I might need to go further as well, will be relying on you. Thanks

**Comment by ShisuiUzumaki** [comment]
> `cd docs; git grep tanh`

@lezcano Which files should I add `softmax` in the `docs` dir
